CHANGELOG.adoc

```diff
@@ -57,6 +57,8 @@ The following methods have been moved to `UtilService` to make them reusable by
 * `ViewNodeService#getAllStandardDoneActions`
 * `ViewNodeService#isAction`
 * `ViewNodeService#isPart`
+- https://github.com/eclipse-syson/syson/issues/687[#687] [services] Fix the drop of an action from the diagram to an action flow compartment.
+The method `ToolService#getMembership` has been renamed `ToolService#moveMembership`, and now returns `void` instead of `Optional<Membership>`.
```

Comment:
No need for this entry because `ToolService#getMembership` was private and so not an API

Reply:
Removed